recipes/tachyon/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd unix
+case "$(uname)" in
+    Darwin) target=macosx-x86_64-thr;;
+    Linux) target=linux-64-thr;;
+esac
+
+make $target -j$CPU_COUNT || make $target -j$CPU_COUNT
```

Comment:
Ah, I forgot to mention. The first time it fails with a permission error, but the 2nd time it passes through. I'm thinking something wrong with the Makefile that makes it the build happen in a wrong order and it fixes itself the second time.